manifests/config.pp

```diff
@@ -40,6 +40,12 @@
           group   => 'root',
           content => template('consul/consul.systemd.erb'),
         }
+        exec { 'systemctl daemon-reload':
```

Comment:
I'm sad we have to do this in general, but is is probable this will conflict with other modules.
https://github.com/camptocamp/puppet-systemd/blob/master/manifests/init.pp#L3
https://github.com/search?utf8=%E2%9C%93&q=%22exec+%7B+%27systemctl+daemon-reload%27%3A%22&type=Code&ref=searchresults

Can you take this approach and make the exec unique to this module?
https://github.com/garethr/garethr-docker/blob/master/manifests/systemd_reload.pp#L6